mloq/config/logic.py

```diff
@@ -44,7 +44,7 @@ def get_docker_image(
     return image
 
 
-def write_config(config: Config, path: Union[Path, str], safe: bool = False):
+def write_config(config: dict, path: Union[Path, str], safe: bool = False):
```

Comment:
Should we always use `OmegaConf` configuration objects in `mloq` internals, or should we allow for arbitrary dictionaries?

Reply:
I changed the signature because we pass in a bare dict - PyCharm highlighted the usage. Ideally